.github/workflows/pyodide.yml

```diff
@@ -54,6 +54,11 @@ jobs:
 
           pip install "setuptools>=70.0.0" PyYAML click packaging pytest
 
+          # First install our patched version of stopit
```

Comment:
Since TimeConstrained is not used, there is no need to install stopit. 

Simpler CI is better, in my opinion.